primitives/linux-port/column.cpp

```diff
@@ -1472,6 +1501,18 @@ void vectorizedFiltering(NewColRequestHeader* in, ColResultHeader* out, const T*
     valuesWritten = vectWriteRIDValues<T, VT, OUTPUT_TYPE, KIND, HAS_INPUT_RIDS>(
         simdProcessor, valuesWritten, validMinMax, ridOffset, dataVecTPtr, ridDstArray, writeMask, Min, Max,
         in, out, nonNullOrEmptyMask, ridArray);
+    if constexpr (KIND != KIND_FLOAT)
```

Comment:
What will happen when ```KIND==KIND_FLOAT```

Reply:
I decided to just add min/max instructions for floats as well, so this branching is removed.